### DISCUSSION TITLE
How to build project?

### DISCUSSION THREAD

**Discussion by nvchernov** [kubernetes-client/csharp#1255] [Q&A]
Hi, I trying to build project but I getting errors, it seems that I should run some code generator or else. Errors about I have no interfaces or classes

![изображение](https://user-images.githubusercontent.com/14893576/228868392-0bb768fd-27d4-4d9f-8a83-7d9ca92ccd69.png)

Thx!!!

**Comment by brendandburns** [comment]
There are lots of examples here:

https://github.com/kubernetes-client/csharp/tree/master/examples

You likely need to include the right nuget package in your build files.

**Comment by tg123** [comment]
dotnet build will generate all stuff you need
